components/bt/host/bluedroid/api/include/api/esp_gap_ble_api.h

```diff
@@ -17,2687 +17,3916 @@
+    } channel_sel_alg;          /*!< ESP_GAP_BLE_CHANNEL_SELECT_ALGORITHM_EVT */
+
     /**
      * @brief ESP_GAP_BLE_PERIODIC_ADV_SYNC_LOST_EVT
+     * Event parameter for the ESP_GAP_BLE_PERIODIC_ADV_SYNC_LOST_EVT event.
+     * This event is triggered when synchronization with a periodic advertiser is lost.
      */
     struct ble_periodic_adv_sync_lost_param {
-        uint16_t sync_handle;                 /*!< sync handle */
-    } periodic_adv_sync_lost;                 /*!< Event parameter of ESP_GAP_BLE_PERIODIC_ADV_SYNC_LOST_EVT */
+        uint16_t sync_handle;              /*!< Handle for the synchronization. Used to identify the synchronization that was lost. */
+    } periodic_adv_sync_lost;    /*!< ESP_GAP_BLE_PERIODIC_ADV_SYNC_LOST_EVT */
+
     /**
      * @brief ESP_GAP_BLE_PERIODIC_ADV_SYNC_ESTAB_EVT
+     * Event parameter for the ESP_GAP_BLE_PERIODIC_ADV_SYNC_ESTAB_EVT event.
+     * This event is triggered when synchronization with a periodic advertiser is established.
      */
     struct ble_periodic_adv_sync_estab_param {
-        uint8_t status;                      /*!< periodic advertising sync status */
-        uint16_t sync_handle;                /*!< periodic advertising sync handle */
-        uint8_t sid;                         /*!< periodic advertising sid */
-        esp_ble_addr_type_t adv_addr_type;   /*!< periodic advertising address type */
-        esp_bd_addr_t adv_addr;              /*!< periodic advertising address */
-        esp_ble_gap_phy_t adv_phy;           /*!< periodic advertising phy type */
-        uint16_t period_adv_interval;        /*!< periodic advertising interval */
-        uint8_t adv_clk_accuracy;            /*!< periodic advertising clock accuracy */
-    } periodic_adv_sync_estab;               /*!< Event parameter of ESP_GAP_BLE_PERIODIC_ADV_SYNC_ESTAB_EVT */
+        uint8_t status;                      /*!< Status of the periodic advertising synchronization. A status value of 0 indicates successful synchronization, while other values indicate errors. */
+        uint16_t sync_handle;                /*!< Handle for the synchronization. Used to identify the established synchronization. */
+        uint8_t sid;                         /*!< Service Identifier for the periodic advertising. Used to identify the type of service. */
+        esp_ble_addr_type_t adv_addr_type;   /*!< Address type of the periodic advertiser. Indicates whether the advertiser's address is public or random. */
+        esp_bd_addr_t adv_addr;              /*!< Address of the periodic advertiser. Used to identify the source of the periodic advertising. */
+        esp_ble_gap_phy_t adv_phy;           /*!< Physical Layer type for the periodic advertising. Indicates the PHY used for advertising. */
+        uint16_t period_adv_interval;        /*!< Interva
```

Comment:
This function used to response the `ESP_GAP_BLE_SEC_REQ_EVT`.
This function may trigger `ESP_GAP_BLE_AUTH_CMPL_EVT`.